sig/transaction/membership.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "transaction",
+  "techLeaders": [
+    {
+      "githubId": "nrc"

```

Comment:
This should be `githubName`: https://github.com/tidb-community-bots/ti-community-bot/pull/75